backend/docs/installation.md

```diff
@@ -0,0 +1,41 @@
+# Installation
+
+1. Fork project on GitHub
+2. Clone project
+    ```bash
+   git clone git@github.com:${YOUR_GITHUB_USER}/urban-league-heat-pump-accelerator.git
+   ```
+3. Install postgres libraries
+   1. MacOS
+        ```bash
+      brew install libpq
+      ```
+   2. Linux (Ubuntu)
+        ```bash
+      sudo apt-get install libpq-dev
+      ```
+4. Install rbenv
+   1. MacOS
+        ```bash
+      brew install rbenv ruby-build \
+      && rbenv init \
+      && source ~/.zshrc \
+      && curl -fsSL https://github.com/rbenv/rbenv-installer/raw/main/bin/rbenv-doctor | bash
+      ```
+   2. Linux (Ubuntu)
+        ```bash
+      sudo apt-get install rbenv \
+      && rbenv init \
+      && source ~/.bashrc \
+      && curl -fsSL https://github.com/rbenv/rbenv-installer/raw/main/bin/rbenv-doctor | bash
+      ```
+5. Install Ruby
+    ```bash
+        cd urban-league-heat-pump-accelerator \
+        && rbenv install \
+        && gem install bundler        
+      ```
+6. Install the backend
+   ```bash
+    cd backend && bundle install
```

Comment:
```suggestion
    cd backend && bin/setup
```

This change will run the setup script that will bundle install and also setup the database.

Reply:
Fixed in commit below